### PR TITLE
 [bitnami/nginx] Remove nginx-ldap-auth-daemon dependency for Bitnami Nginx Chart

### PR DESCRIPTION
**Pull request by mdhont**
Signed-off-by: michield <michield@vmware.com>

Description of the change

Remove nginx-ldap-auth-daemon dependency from Bitnami Nginx Charts.

### CODE REVIEW

**Comment by dgomezleon** [comment]
LGTM

**Review by dgomezleon — DISMISSED** [review]
LGTM

**Review by carrodher — COMMENTED** [review]
I would add a note in the Upgrade section describing what is the reason behind a major version bump, see https://github.com/bitnami/charts/tree/master/bitnami/nginx#upgrading

**Review by carrodher — APPROVED** [review]
LGTM

File: bitnami/nginx/templates/server-block-configmap.yaml [hunk 1]
```diff
@@ -5,7 +5,6 @@ metadata:
   labels: {{- include "common.labels.standard" . | nindent 4 }}
 data:
   server-blocks-paths.conf: |-
-    include  "/opt/bitnami/nginx/conf/server_blocks/ldap/*.conf";
     include  "/opt/bitnami/nginx/conf/server_blocks/common/*.conf";
```
Comment (joancafom) [inline bitnami/nginx/templates/server-block-configmap.yaml:8]: This should be removed.

**Review by joancafom — CHANGES_REQUESTED** [review]
Thanks for your PR! Could you please take a look at my comments?

**Review by joancafom — APPROVED** [review]
LGTM!

**Review by carrodher — DISMISSED** [review]

**Comment by carrodher** [comment]
Please note the deprecation of the `bitnami/nginx-ldap-auth-daemon` container was planned some time ago and it is not related to any security issue. We have a policy of not releasing on our side software that is not maintained by the upstream project (or the release cadence is not frequent).

The latest release in the upstream project (https://github.com/nginxinc/nginx-ldap-auth) dates from 31 Oct 2019 which doesn't meet the previously mentioned policy hence the deprecation process was triggered on our side **at the end of February**.

In this case, a deprecation notice was added to the container README on 4th March (https://github.com/bitnami/bitnami-docker-nginx-ldap-auth-daemon/commit/7737d2a3080f2dc060fc356d52c1c09fbbdb6b48):

> ## Deprecation Notice
> 
> NGINX LDAP Auth daemon is no longer maintained by upstream and is now internally tagged as to be deprecated. This image will no longer be released in our catalog a month after this notice is published, but already released container images will still persist in the registries. Valid to be removed starting on: 03-25-2022

After this period, the deprecation was executed and the container was removed from the Bitnami NGINX Helm chart (https://github.com/bitnami/charts/pull/9645), as well the [container repository](https://github.com/bitnami/bitnami-docker-nginx-ldap-auth-daemon) itself was marked as archived with the following [note in the README](https://github.com/bitnami/bitnami-docker-nginx-ldap-auth-daemon#deprecation-notice):

> # DEPRECATION NOTICE

> Since the upstream project is not maintained, this image has been deprecated, and will no longer be maintained nor updated.

**Pull request merged**